felix/dataplane/linux/vxlan_mgr.go

```diff
@@ -692,19 +692,19 @@ func (m *vxlanManager) configureVXLANDevice(mtu int, localVTEP *proto.VXLANTunne
 
 	// Make sure the IP address is configured.
 	if err := m.ensureAddressOnLink(addr, link); err != nil {
-		return fmt.Errorf("failed to ensure address of interface: %s", err)
+		return fmt.Errorf("failed to ensure address of interface: %v", err)
```

Comment:
in fact it's not. We get the log as mentioned in pr description.

Reply:
The log in the description is from line 588, so I think is different.

This example seems to suggest that %s is OK: https://go.dev/play/p/DDXdumlQW3Q

That said, using `%v` will also generate the same output, I just like to avoid making changes that aren't needed as it complicates the git blame history.